php/blocks/class-loader.php

```diff
@@ -250,6 +250,18 @@ public function render_block_template( $block, $attributes ) {
 		$block_lab_attributes = $attributes;
 		$block_lab_config     = $block;
 
+		/**
+		 * Runs in the 'render_callback' of the block.
+		 *
+		 * For example, if a block depends on a JavaScript file,
+		 * this is a good place to call wp_enqueue_script().
+		 * In that case, pass true as the 5th argument ($in_footer) to wp_enqueue_script().
+		 *
+		 * @param array $block The block that is rendered.
+		 * @param array $attributes The block attributes.
+		 */
+		do_action( 'block_lab_render_block_template', $block, $attributes );
```

Comment:
Should we also create a second action specifically for the current block? e.g.

```
do_action( "block_lab_render_block_template_{$block['name']}", $block, $attributes );
```

Reply:
Good question. It might depend on if scripts will need to be enqueued for several blocks. 

If a script is only needed for one block, that dynamic action above would definitely be simpler.

Reply:
I think I'm suggesting we do both.

Reply:
Ah, OK.